src/app/modules/account/pages/registration/components/user-registration/user-registration.component.html

```diff
@@ -15,9 +15,90 @@
 GNU Affero General Public License for more details.
 
 You should have received a copy of the GNU Affero General Public License
-along with Nonprofit Social Networking Platform.  If not, see <https://www.gnu.org/licenses/>.
--->
-<p>
-  user-registration works!
-  {{account?.name}}
-</p>
\ No newline at end of file
+along with Nonprofit Social Networking Platform.  If not, see <https://www.gnu.org/licenses/>.-->
+
+<form [formGroup]="registrationForm" (ngSubmit)="onSubmit()">
+  <ion-grid>
+    <ion-row>
+      <ion-col>
+
+        <ion-input class="custom-ion-item" formControlName="name" type="text" label="Name" label-placement="floating"
+          fill="outline" placeholder="Enter Name"></ion-input>
+
+        <ion-input class="custom-ion-item" formControlName="tagline" type="text" label="Tagline"
+          label-placement="floating" fill="outline" placeholder="Enter Tagline"></ion-input>
+
+
+        <ion-input class="custom-ion-item" type="text" label="Description" label-placement="floating" fill="outline"
+          placeholder="Enter Description" formControlName="description"></ion-input>
+
+
+        <ion-input class="custom-ion-item" placeholder="888-888-8888"  type="tel" label="Phone Number"
```

Comment:
For the `phone #` fields can you have a look at the `edit-user-profile` files and see if you can do something similar.